samples/subsys/usb/console/nrf52840_pca10056.overlay

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright (c) 2018 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+/ {
```

Comment:
put empty line here like in all other overlays

Reply:
@finikorg thats part of the problem, you need to find the .dts file that describes your board and add that usb_cdc: virtualcom block otherwise the overlay doesn't work. I personally found that to be quite confusing, as all I really wanted to do was in my zephyr app set the uart console device name to the usb cdc uart device name. But there were hoops to be jumped through.

Reply:
@finikorg this file is a direct copy of the arduino101_overlay.dts
https://github.com/zephyrproject-rtos/zephyr/blob/master/samples/subsys/usb/console/arduino_101.overlay#L1

Reply:
> @finikorg this file is a direct copy of the arduino101_overlay.dts
> https://github.com/zephyrproject-rtos/zephyr/blob/master/samples/subsys/usb/console/arduino_101.overlay#L1

OK, missed with tinytile and quark_se.